bin/uglifyjs

```diff
@@ -273,6 +273,9 @@ if (ARGS.comments != null) {
 
 var files = ARGS._.slice();
 
+if (process.platform === "win32")
+    files = UglifyJS.simple_glob(files);
```

Comment:
This patch works fine on Mac if I comment out line 276. Just need to verify it works on Windows.